File: app/src/main/java/com/shevelev/comics_viewer/activities/comics_creation/SortPagesActivityView.java
package com.shevelev.comics_viewer.activities.comics_creation;

import android.content.Context;
import android.widget.RelativeLayout;

import com.shevelev.comics_viewer.common.structs.Size;

/**
 * Contecst view for ComicsCreationActivity
 */
public class SortPagesActivityView extends RelativeLayout
{
    public SortPagesActivityView(Context context, int layoutId)
    {
        super(context);

        inflate(context, layoutId, this);
    }

    public Size getSize()
    {
        return new Size(getWidth(), getHeight());
    }
}

File: app/src/main/java/com/shevelev/comics_viewer/activities/comics_creation/SortPagesActivityView.kt
package com.shevelev.comics_viewer.activities.comics_creation

import android.content.Context
import android.view.View
import android.widget.RelativeLayout
import com.shevelev.comics_viewer.common.structs.Size

/**
 * Context view for ComicsCreationActivity
 */
class SortPagesActivityView(context: Context?, layoutId: Int) : RelativeLayout(context) {
    val size: Size
        get() = Size(width, height)

    init {
        View.inflate(context, layoutId, this)
    }
}